specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/examples/IpTagsCreate.json

```diff
@@ -0,0 +1,66 @@
+{
+    "parameters": {
+      "api-version": "2019-07-01",
+      "subscriptionId": "subId",
+      "resourceGroupName": "myResourceGroup",
+      "ipTagName": "ipTags1",
+      "parameters": {
+        "tags": {
```

Comment:
the parameters should contain the ipAddresses array

Reply:
Thanks. Fixed